### ISSUE TITLE
I can't export the artboard with the gif playing

### ISSUE THREAD

**Issue by adop31** [kannonboy/sketch-gifme-plugin#15]
Hello to all! I can't export the arboard with the GIF inserted...  It doesn't exist the option to export in .gif one only artboard? When I upload the artboard to Invision by Craft, it doesnt works too... Do you know the solution?

If I use another plugin called " Export more" it's also imposible... It only for create Gifs with diferents artboards ;)

**Comment by eliasjulian** [comment]
This would be a really great feature, but unfortunately it's not something we currently have built into the plugin. I'm honestly not quite sure how it would be accomplished or if it's even possible. 

Essentially what happens when you run the plugin and add a .gif to your sketch file is ffmpeg breaks out every frame into a layer fill and sketch is rapidly changing the fill, essentially "playing" the gif, but the native file doesn't exist as a whole in your file, just the separate frames. So to export it along with the rest of your artboard we'd have to re-piece them together as a new .gif file based on all the rapidly changing layer fills + your other sketch elements on the artboard.

Open to commits if you have an idea for how to technically accomplish that!

**Comment by Potatomaker** [comment]
Yes there is. What you need to do is make slices from each artboard. Then using the Save for web legacy feature, you can export each slice individually. Hope this helps.
